inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -167,11 +180,22 @@ ExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadExeNetworkImpl(co
     DeviceMap<SoExecutableNetworkInternal> executableNetworkPerDevice;
     std::mutex load_mutex;
     std::vector<Task> loads;
+    std::once_flag readNetworkFlag;
     for (auto& p : metaDevices) {
         loads.push_back([&]() {
             const auto &deviceName = p.deviceName;
             const auto &deviceConfig = p.config;
-            auto exec_net = GetCore()->LoadNetwork(network, deviceName, deviceConfig);
+            SoExecutableNetworkInternal exec_net;
+            if (modelPath.empty()) {
+                exec_net = GetCore()->LoadNetwork(network, deviceName, deviceConfig);
```

Comment:
@ilya-lavrenov just wondering whether LoadNEtwork(filename) would fallback to the conventional Read+Load, or fail if the Import fails